Automattic/sensei
# Refactor video extension feature removing wrapper

Fixes #5418
Closes #5491

### Changes proposed in this Pull Request

* Refactor video extension feature to remove wrapper that caused style issues (Now it uses the classes from video and embed block for the selectors).
* It also uses the player class to have a simplified code in the video extensions.

### Testing instructions


* Create a course with a lesson.
* In the course sidebar, check all video settings ("Autocomplete Lesson", "Autopause", and "Required").
* In the lesson, add a video for each type (Video, VideoPress, YouTube, and Vimeo).
* Enroll to the course, and access the lesson.
* Play all the videos, and change the browser tab.
* Make sure all the videos were paused.
* Play all the videos until the end (you can just move the video position to the end while playing).
* Make sure that every completed video, it updates the complete lesson button counter.
* Also, make sure that after completing the videos, it autocompletes the lesson.
